### PR TITLE
Generate rejections via ProtoData

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR migrates code generation for rejection `Throwable`s to be done via a ProtoData plugin.

Previously, the code generation was based on an internal Gradle plugin which was added to a project by McJava. Now, the code is generated via a custom `JavaRenderer`, which is passed as ProtoData plugin together with one for Validation.

### Breaking changes
  * The code of rejection throwables now is placed under the `generated/main/java` directory. Before this PR it was placed under `generated/main/spine`.

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@dmdashenkov, both locally and at GitHub CI the `Test build performance` check reports this:

```
> Task :checkPerformance FAILED
> :checkPerformance
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkPerformance'.
> process hasn't exited
```

Could you look into why it does not finish? You know, finishing is a good thing... ;)

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, there are known performance problems in `Validation`. There is not going to be a quick fix for this. I'm currently working on this issue.

**Comment by alexander-yevsyukov** [comment]
> @alexander-yevsyukov, there are known performance problems in `Validation`. There is not going to be a quick fix for this. I'm currently working on this issue.

@dmdashenkov, we increased the timeout for the `checkPerformance` task and it passed.

File: version.gradle.kts [hunk 1]
```diff
@@ -32,5 +32,5 @@
  * For versions of Spine-based dependencies please see [io.spine.internal.dependency.Spine].
  * Keep in mind that changind it under `buildSrc` also requires sync. with `tests/buildSrc`.
  */
-val mcJavaVersion by extra("2.0.0-SNAPSHOT.153")
+val mcJavaVersion by extra("2.0.0-SNAPSHOT.160")
```
Comment (dmdashenkov) [inline version.gradle.kts:35]: Semver is dead, long live semver.

**Review by dmdashenkov — APPROVED** [review]
LGTM.

File: version.gradle.kts (hunk 1 above)
Comment (alexander-yevsyukov) [reply version.gradle.kts:35]: We introduce a breaking change in this PR. So, I bumped the version to the next dozen to highlight the fact.

**Review by armiol — APPROVED** [review]

**Pull request merged**